Update FAQs in faq.md

avoidthehack opened the pull request:
Clarified existing FAQs and added new FAQs. Linked out to key pages on the main LinkStack website.

avoidthehack committed:
Update faq.md

Clarified exist FAQs and added new FAQs. Linked out to key pages on the main LinkStack website.

MagicLike commented on docs/faq.md:
```diff
@@ -2,26 +2,47 @@
 ## What do I need to use LinkStack?
 
-To host your own instance of LinkStack, you will need a web server running at least PHP 8.0.
+To host your own instance of LinkStack, you will need a web server running at least PHP 8.1
```
The full stop is missing here

MagicLike commented on docs/faq.md:
```diff
@@ -2,26 +2,47 @@
 ## What do I need to use LinkStack?
 
-To host your own instance of LinkStack, you will need a web server running at least PHP 8.0.
+To host your own instance of LinkStack, you will need a web server running at least PHP 8.1
```
The full stop is missing here

MagicLike replied:
still missing ^^

avoidthehack replied:
I added and changed it back to 8.0 (per a conversation with the main maintainer last month.)

I also linked out to the newly proposed access admin page.

MagicLike commented on docs/faq.md:
```diff
@@ -2,26 +2,47 @@
+
+Self-hosting is not the only option available for using LinkStack. 
+
+The project offers various tiers of [hosted instances](https://linkstack.org/hosted/) for GitHub Sponsors. Additionally, you can find [instances of LinkStack](https://linkstack.org/instances/) hosted by others in the community.
```
Please also remove the trailing slashes ^^

MagicLike reviewed (changes requested):
LGTM, there are only a few small corrections to be made :+1:

avoidthehack committed:
Update faq.md

avoidthehack committed:
Update faq.md

Added period after 8.1 (and changed back to 8.0, per information given by the developers).

Also added section for the admin area and linking out to the newly created Access admin page.
